src/main/java/ui/components/pickers/MilestonePickerState.java

```diff
@@ -273,6 +276,18 @@ private void addMatchingMilestones(List<PickerMilestone> milestonesToBePopulated
                 .findFirst();
     }
 
+    /**
+     * Returns the list of milestones that the user is allowed to select.
```

Comment:
Shouldn't mention implementation detail such as `the list` as it might change to `set` or other things. `Returns milestones that ...` is sufficient.